services/google_ai_service.py

```diff
@@ -0,0 +1,26 @@
+from services.ai_service import AIService
+import openai
+import os
+
+# To use Google Cloud's AI products, you'll need to install Google Cloud CLI and enable the TTS and in your project: https://cloud.google.com/sdk/docs/install
+from google.cloud import texttospeech
+
+class GoogleAIService(AIService):
```

Comment:
Just to clarify - does this demo currently implemented entry points into all of these different services? (i.e., is it possible for this to actually be used in practice atm?)